RevenueCatUI/CustomerCenter/Views/FeedbackSurveyView.swift

```diff
@@ -33,19 +33,27 @@ struct FeedbackSurveyView: View {
     @Environment(\.colorScheme)
     private var colorScheme
 
+    @Binding
+    private var isPresented: Bool
```

Comment:
Hmm I'm a bit confused, this seems to only be assigned but not read?

Reply:
this might be a bit confusing yes... I create a binding here https://github.com/RevenueCat/purchases-ios/pull/4444/files#diff-121dcc7927012032acdbbf026716bf9daf66d91cee530ab12eae052d06526aa0R68

```
.isNotNil(self.$viewModel.feedbackSurveyData)
```

So if I set that to false, it nullifies the `feedbackSurveyData` which hides the feedback survey, since it's only presented if `feedbackSurveyData` is not nil.

Another approach I can use is adding an `onDismiss` callback:

```
.navigationDestination(isPresented: .isNotNil(self.$viewModel.feedbackSurveyData)) {
      if let feedbackSurveyData = self.viewModel.feedbackSurveyData {
          FeedbackSurveyView(feedbackSurveyData: feedbackSurveyData,
                             customerCenterActionHandler: self.customerCenterActionHandler,
                             onDismiss: { self.viewModel.feedbackSurveyData = nil })
      }
  }
```

Maybe that's more clear?

Reply:
Ahhh got it! Well, I think it might be clearer, but I think it's ok as is. Thanks!